generation/WinSDK/enums.json

```diff
@@ -30632,5 +30669,136 @@
         "field": "dwFlags"
       }
     ]
+  },
+  {
+    "name": "BRUSH_STYLE",
+    "members": [
+      {
+        "name": "BS_SOLID",
+        "value": 0
+      },
+      {
+        "name": "BS_NULL",
+        "value": 1
+      },
+      {
+        "name": "BS_HOLLOW",
+        "value": "BS_NULL"
+      },
+      {
+        "name": "BS_HATCHED",
+        "value": 2
+      },
+      {
+        "name": "BS_PATTERN",
+        "value": 3
+      },
+      {
+        "name": "BS_INDEXED",
+        "value": 4
+      },
+      {
+        "name": "BS_DIBPATTERN",
+        "value": 5
+      },
+      {
+        "name": "BS_DIBPATTERNPT",
+        "value": 6
+      },
+      {
+        "name": "BS_PATTERN8X8",
+        "value": 7
+      },
+      {
+        "name": "BS_DIBPATTERN8X8",
+        "value": 8
+      },
+      {
+        "name": "BS_MONOPATTERN",
+        "value": 9
+      }
+    ],
+    "uses": [
+      {
+        "struct": "tagLOGBRUSH",
+        "field": "lbStyle"
+      },
+      {
+        "struct": "tagLOGBRUSH32",
+        "field": "lbStyle"
+      }
+    ]
+  },
+  {
+    "name": "TMPF_FLAGS",
+    "flags": "true",
```

Comment:
This is enum is not a flag. Notice the values increase numerically and aren't meant to be or'ed together.

Reply:
Ah - my initial values were incorrect! They are flags but they should be the binary values of 1, 2, 4, 8. Changes should be reflected in my latest push.